bin/sozo/src/commands/execute.rs

```diff
@@ -20,6 +20,28 @@ use crate::utils;
 #[derive(Debug, Args)]
 #[command(about = "Execute a system with the given calldata.")]
 pub struct ExecuteArgs {
+
+    #[arg(
+        help = "The calls to be executed. Each call should include the address or tag, entrypoint, and calldata."
+    )]
+
+    pub calls: Vec<CallArgs>,
+
+    #[command(flatten)]
+    pub starknet: StarknetOptions,
+
+    #[command(flatten)]
+    pub account: AccountOptions,
+
+    #[command(flatten)]
+    pub world: WorldOptions,
+
+    #[command(flatten)]
+    pub transaction: TransactionOptions,
+}
+
+#[derive(Debug, Args)]
+pub struct CallArgs {
```

Comment:
Here you'll have an issue since you don't have any way to parse those data from the CLI.
You will need to find a way to define those multi calls in a parseable manner in order to have clap parsing them.

Accepting a `Vec<String>` and then parsing the calls is a good way to go. Or if you have a `CallArgs` type, then this one must be parseable from `String`.

Reply:
@glihm , in the latest commit I have kept Vec<String> as you have suggested, and then I am converting this string into CallArgs type for further processing.

Reply:
Also, let me know if you have any problems with current implementation.